encoders/numeric/TruncatedSVDEncoder/README.md

```diff
@@ -0,0 +1,13 @@
+# TruncatedSVDEncoder
+
+`TruncatedSVDEncoder` encodes data from an `ndarray` of size `BatchSize x Dimension` into an `ndarray` of size `BatchSize x EmbeddingDimension` using [Truncated Singular Value Decomposition (SVD)](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.TruncatedSVD.html).  Truncated SVD is a dimension reduction technique which does not center the data before computation which makes it efficient for sparse matrices.
```

Comment:
It is not clear, does it work with sparse input or sparse output? In any case, it is not clear in documentation or method signatures

Reply:
From the documentation and the examples, it works and can operate on sparse inputs. Should I add that in the documentation/signatures?